goldstone/core/views.py

```diff
@@ -518,3 +529,89 @@ def get(self, request, uuid, *args, **kwargs):
 
         else:
             return Response({}, status=HTTP_404_NOT_FOUND)
+
+
+###############
+# Event views #
+###############
+
+# Our API documentation extracts this docstring, hence the use of markup.
+class EventSummarizeView(ElasticListAPIView):
+    """Return an aggregation summary of events from Logstash data.
+
+    ---
+
+    GET:
+        parameters:
+           - name: timestamp__range
+             description: The time range, as {'xxx':nnn}. Xxx is gte, gt, lte,
+                          or lt.  Nnn is an epoch number.  E.g.,
+                          {'gte':1430164651890}. You can also use AND, e.g.,
+                          {'gte':1430164651890, 'lt':1455160000000}
+             paramType: query
+           - name: interval
+             description: The desired time interval, as n(s|m|h|w). E.g., 1d
+                          or 3m.
+             paramType: query
+
+    """
+
+    serializer_class = LogEventAggSerializer
```

Comment:
LogEventAggSerializer will probably disappear when we remove glogging Events.  Might be worth ticketing it for refactor or doing it now.